modules/cluster-logging-loki-about.adoc

```diff
@@ -0,0 +1,41 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: CONCEPT
+[id="about-logging-loki_{context}"]
+= About the LokiStack
+
+In {logging} documentation, *LokiStack* refers to the {logging} supported combination of Loki, and web proxy with {product} authentication integration. LokiStack's proxy uses {product} authentication to enforce multi-tenancy. *Loki* refers to the log store as either the individual component or an external store.
```

Comment:
`{product}` isn't a valid attribute and renders as `{product}` in the preview. Do you mean `{product-title}` so that it renders "OpenShift Container Platform"?

Reply:
Fixed, thank you.